src/v/config/configuration.cc

```diff
@@ -3174,7 +3174,15 @@ configuration::configuration()
       "Minimum number of active producers per virtual cluster",
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
       std::numeric_limits<uint64_t>::max(),
-      {.min = 1}) {}
+      {.min = 1})
+  , unsafe_enable_consumer_offsets_delete_retention(
+      *this,
+      "unsafe_enable_consumer_offsets_delete_retention",
+      "Enables delete retention of consumer offsets topic when. This is an "
```

Comment:
Enables delete retention of consumer offsets topic when...when what?

Reply:
yikes, fixed.